### PR TITLE
Automatically print inner comments

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

This PR updates `@babel/generator` to automatically print `.innerComments` whenever there are two contiguous "raw" tokens (i.e. not nested nodes). By doing so, we reduce the risk of accidentally forgetting to print inner comments.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/15080"></a>

### CODE REVIEW

**Review by JLHwung — APPROVED** [review]

File: packages/babel-generator/test/fixtures/comments/import-reflection-inner-comment/output.js [hunk 1]
```diff
@@ -1 +1 @@
-/* 0 */import /* 1 */module /* 2 */from /* 3 */ from /* 4 */"./module.wasm" /* 5 */;
\ No newline at end of file
+/* 0 */import /* 1 */ module /* 2 */from /* 3 */ from /* 4 */"./module.wasm" /* 5 */;
```
Comment (liuxingbaoyu) [inline packages/babel-generator/test/fixtures/comments/import-reflection-inner-comment/output.js:1]: There is an inconsistency in the spaces here, but there have been similar ones in the past, and ignoring them should be fine.

File: packages/babel-generator/test/fixtures/comments/for-await-using-of-comments/output.js [hunk 2]
```diff
@@ -1 +1 @@
-/*1*/for await /* 2 */ /*3*/( /*4*/using /*5*/fo /*6*/ of /*7*/of /*8*/) /*9*/;
\ No newline at end of file
+/*1*/for /* 2 */ /*3*/ await ( /*4*/using /*5*/fo /*6*/ of /*7*/of /*8*/) /*9*/;
```
Comment (liuxingbaoyu) [inline packages/babel-generator/test/fixtures/comments/for-await-using-of-comments/output.js:1]: The original behavior is better for me, but I don't mind the new one.

File: packages/babel-generator/test/fixtures/comments/async-class-method-computed-key-comment/output.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
 class C {
-  async [/* 2 */foo /* 3 */] /* 1 */ /* 4 */ /* 5 */() {}
+  async /* 1 */ /* 4 */ /* 5 */ [/* 2 */foo /* 3 */]() {}
```
Comment (liuxingbaoyu) [inline packages/babel-generator/test/fixtures/comments/async-class-method-computed-key-comment/output.js:2]: The original behavior is better for me, but I don't mind the new one.

**Review by liuxingbaoyu — APPROVED** [review]
Great!

File: packages/babel-generator/test/fixtures/comments/import-reflection-inner-comment/output.js (hunk 1 above)
Comment (nicolo-ribaudo) [reply packages/babel-generator/test/fixtures/comments/import-reflection-inner-comment/output.js:1]: Yeah I tried to fix some of them, but they have different causes and I couldn't figure out how ti fix _all_ of them :sweat_smile:

File: packages/babel-generator/test/fixtures/comments/async-class-method-computed-key-comment/output.js (hunk 3 above)
Comment (nicolo-ribaudo) [reply packages/babel-generator/test/fixtures/comments/async-class-method-computed-key-comment/output.js:2]: Yeah the problem is that there is no "correct" answer: due to how we mix inner comments, two of 1/4/5 will be in the wrong place.

**Pull request merged**